astropy/io/fits/tests/test_core.py

```diff
@@ -593,6 +595,48 @@ def test_open_nonexistent(self):
             assert os.path.exists(self.temp('foobar.fits'))
             os.remove(self.temp('foobar.fits'))
 
+    @pytest.mark.skipif(six.PY2,
```

Comment:
Ok for skipping the test, but in this case it would be hazardous to backport this change, so maybe we just re-milestone this to 3.0 ?